tests/output-eve-ftp/suricata.yaml

```diff
@@ -0,0 +1,9 @@
+%YAML 1.1
```

Comment:
why is this needed? The default config has ftp enabled.

Reply:
Will remove.